tests/unit/htdocs/modules/protector/ProtectorMysqlDatabaseTest.php

```
@@ -224,4 +224,93 @@ public function doubtfulRequestsCanBeSet(): void
+    // ---------------------------------------------------------------
+
+    #[Test]
+    public function execMethodExists(): void
+    {
+        $this->assertTrue(method_exists($this->db, 'exec'));
+    }
+
+    #[Test]
+    public function execMethodIsDeclaredOnProtectorClass(): void
+    {
+        $ref = new \ReflectionMethod(\ProtectorMySQLDatabase::class, 'exec');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '241', column '20'). (undefined)

(MissingImport)